helm/single-content/Chart.yaml

```diff
@@ -3,13 +3,13 @@
 # END COPYRIGHT NOTICE
 apiVersion: v2
 name: single-content
-version: 0.4.0
+version: 0.5.0
 appVersion: "24.1"
 description: A configurable single-IDOL setup
 dependencies:
 - name: idol-library
-  version: "0.4.0"
-  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
+  version: "0.5.0"
+  repository: file://../idol-library
```

Comment:
Shouldn't commit this line (once 0.5.0 release .tgz is published)?

Reply:
[c16dfcd](https://github.com/opentext-idol/idol-containers-toolkit/pull/30/commits/c16dfcd28d372844173847fef5fb5375c3cf3508)